packages/types/farFU.cfc

```diff
@@ -806,10 +806,9 @@
 				<cfif not isvalid("integer",stResult.__redirectionType)>
```

Comment:
I have removed the <cftry> tag now. Also did some cleanup of indenting and white space.